packages/suite/src/components/recovery/SelectWordCount.tsx

```diff
@@ -1,50 +1,44 @@
 import React from 'react';
-
-import { P } from '@trezor/components';
-import { Option, Wrapper } from '@onboarding-components';
+import styled from 'styled-components';
+import { NeueOption, OptionsWrapper, OptionsDivider } from '@onboarding-components';
 
 import { WordCount } from '@recovery-types';
 import { Translation } from '@suite-components';
 
+const StyledOption = styled(NeueOption)`
+    justify-content: center;
+`;
+
 interface Props {
     onSelect: (number: WordCount) => void;
 }
 
 const SelectWordCount = ({ onSelect }: Props) => (
-    <>
-        <P size="small">
-            <Translation id="TR_RECOVER_SUBHEADING" />
-        </P>
-        <Wrapper.Options>
-            <Option
-                variant={3}
-                action={() => {
-                    onSelect(12);
-                }}
-                button={<Translation id="TR_WORDS" values={{ count: '12' }} />}
-                imgSrc="images/svg/12-words.svg"
-                data-test="@recover/select-count/12"
-            />
-            <Option
-                variant={3}
-                action={() => {
-                    onSelect(18);
-                }}
-                button={<Translation id="TR_WORDS" values={{ count: '18' }} />}
-                imgSrc="images/svg/18-words.svg"
-                data-test="@recover/select-count/18"
-            />
-            <Option
-                variant={3}
-                action={() => {
-                    onSelect(24);
-                }}
-                button={<Translation id="TR_WORDS" values={{ count: '24' }} />}
-                imgSrc="images/svg/24-words.svg"
-                data-test="@recover/select-count/24"
-            />
-        </Wrapper.Options>
-    </>
+    <OptionsWrapper>
+        <StyledOption
+            onClick={() => {
+                onSelect(12);
+            }}
+            heading={<Translation id="TR_WORDS" values={{ count: '12' }} />}
+            data-test="@recover/select-count/12"
+        />
+        <OptionsDivider />
+        <StyledOption
+            onClick={() => {
+                onSelect(18);
+            }}
+            heading={<Translation id="TR_WORDS" values={{ count: '18' }} />}
+            data-test="@recover/select-count/18"
+        />
+        <OptionsDivider />
+        <StyledOption
+            onClick={() => {
+                onSelect(24);
+            }}
+            heading={<Translation id="TR_WORDS" values={{ count: '24' }} />}
+            data-test="@recover/select-count/24"
+        />
+    </OptionsWrapper>
```

Comment:
There's some repetition here, can be looped.

Reply:
I'm sorry, but I don't really see much of a point this. these are literally 3 static components separated by divider, there won't be more of them, there is no strong usecase for turning it into an array of items and them looping over them and handle rendering divider just in between. huge overengineering imo.